proposals/spi-proposal.md

```diff
@@ -92,13 +89,17 @@ of the transfer and then set to HIGH at the end of the transfer.
 9:  maxSpeed              (bits 29 - 32)
 10: wordSize              (0 = DEFAULT = 8-bits, 1 = 1-bit, 2 = 2 bits, etc)
 11: csPin                 [optional] (0-127) The chip select pin number
-12: csActiveState         (required if csPin specified) 0 = Active LOW, 1 = Active HIGH
+12: csPinOptions          (required if csPin specified)
```

Comment:
I think this works well. It's optional and handles the basic cases for CS control (except for multiple transfers).